crates/tests/standalone/build.rs

```diff
@@ -165,7 +165,14 @@ fn write_no_inner_attr(output: &str, filter: &[&str]) {
 }
 
 fn riddle(output: &str, filter: &[&str], config: &[&str]) {
-    _ = std::fs::remove_file(output);
+    // Rust-analyzer may re-run build scripts whenever a source file is deleted
+    // which causes an endless loop if the file is deleted from a build script.
+    // To workaroud this, we truncate the file instead of deleting it.
+    // See #2777.
```

Comment:
```suggestion
    // See https://github.com/microsoft/windows-rs/issues/2777.
```

Just makes it easier to open the issue in VS Code.

Reply:
Good point!